examples/text_to_image/train_text_to_image.py

```diff
@@ -797,15 +838,16 @@ def collate_fn(examples):
             args.resume_from_checkpoint = None
         else:
             accelerator.print(f"Resuming from checkpoint {path}")
-            accelerator.load_state(os.path.join(args.output_dir, path))
-            global_step = int(path.split("-")[1])
```

Comment:
Why was this a problem here? For me this logic was pretty easy to undrestand and didn't require us to nest checkpoints in another checkpoint folder